src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (people) => {
+  return (
+    people.reduce((prev, person) => person.died - person.born + prev, 0)
+    / people.length
+  );
+};
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let filteredPeople = people.filter((person) => person.sex === 'm');
```

Comment:
I guess it's not really good name, what if we have to create an additional variable for women?

Reply:
agree, bad variable name

Reply:
and let's create a helper function for filtering people by gender